### PR TITLE
Add related articles to publication detail page (DI-1147)

### PR DESCRIPTION
**Pull request by alimpens**

### CODE REVIEW

File: src/app/pages/PublicationDetailPage/PublicationDetailPage.jsx [hunk 1]
```diff
@@ -6,21 +6,37 @@ import {
...
 import ContentContainer from '../../components/ContentContainer/ContentContainer'
 import DocumentCover from '../../components/DocumentCover/DocumentCover'
 import EditorialPage from '../../components/EditorialPage/EditorialPage'
+import EditorialResults from '../../components/EditorialResults'
 import ShareBar from '../../components/ShareBar/ShareBar'
 import getImageFromCms from '../../utils/getImageFromCms'
 import useDownload from '../../utils/useDownload'
 import useFromCMS from '../../utils/useFromCMS'
 
+const Divider = styled.div`
```
Comment (jonkoops) [inline src/app/pages/PublicationDetailPage/PublicationDetailPage.jsx:29]: We might have a divider like component in ASC but not sure of that. Note that a [hr](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/hr) tag _might_ be more appropriate.

File: src/app/pages/PublicationDetailPage/PublicationDetailPage.jsx [hunk 2]
```diff
@@ -6,21 +6,37 @@ import {
...
+  width: 200px;
+  height: 3px;
+  background-color: ${themeColor('secondary')};
+  margin: ${themeSpacing(8, 0, 6)};
+`
+
+const StyledEditorialResults = styled(EditorialResults)`
+  margin-bottom: ${themeSpacing(25)};
```
Comment (jonkoops) [inline src/app/pages/PublicationDetailPage/PublicationDetailPage.jsx:37]: That's a lot of margin hahah

File: src/app/pages/PublicationDetailPage/PublicationDetailPage.jsx [hunk 3]
```diff
@@ -99,6 +116,20 @@ const PublicationDetailPage = () => {
                     {body && <CustomHTMLBlock body={body} />}
                   </EditorialContent>
                 </Column>
+
+                {related && related.length ? (
```
Comment (jonkoops) [inline src/app/pages/PublicationDetailPage/PublicationDetailPage.jsx:120]: You could use optional chaining here.
```suggestion
                {related?.length ? (
```

**Review by jonkoops — APPROVED** [review]

File: src/app/pages/PublicationDetailPage/PublicationDetailPage.jsx (hunk 1 above)
Comment (alimpens) [reply src/app/pages/PublicationDetailPage/PublicationDetailPage.jsx]: I can't find it in ASC... I agree that an hr tag is more appropriate, the divider represents a non-arbitrary seperation of content.

File: src/app/pages/PublicationDetailPage/PublicationDetailPage.jsx (hunk 2 above)
Comment (alimpens) [reply src/app/pages/PublicationDetailPage/PublicationDetailPage.jsx:37]: It's huge :) It's the same in the design for related articles in the ArticleDetailPage

File: src/app/pages/PublicationDetailPage/PublicationDetailPage.jsx (hunk 3 above)
Comment (alimpens) [reply src/app/pages/PublicationDetailPage/PublicationDetailPage.jsx]: Good one, done!

**Review by jonkoops — APPROVED** [review]

**Comment by jonkoops** [comment]
Looks good to me, don't forget to squash the commit when merging.

**Pull request merged**